packages/yoga/src/Tag/native/Informative.jsx

```diff
@@ -58,14 +58,17 @@ const TagInformative = ({
       components: { tag },
     },
   },
+  size,
   ...props
 }) => (
-  <Informative {...props}>
+  <Informative size={size} {...props}>
```

Comment:
The Tag.Informative should follow the same pattern as Buttons do, using `small` as a boolean value.
We won't have more than two sizes for Tag and, if this would be a thing, we could come back and rethink/refactor.

Reply:
Done! @ggdaltoso